lambdas/navigator_config/index.py

```diff
@@ -63,14 +80,55 @@ def config(dest_bucket, dest_dir, es_url, api_url, s3_bucket,
             'accessKeyId': '',
             'secretAccessKey': ''
         },
+        'registryUrl': 'https://' + registry_host + '/',
```

Comment:
Might want to be careful about possible trailing slash in case user enters that as a parameter to the CFT.

Reply:
Would it be an acceptable alternative to document in the parameter that there should be no leading or trailing slashes?

Reply:
Sure, looks like we might be able to set `AllowPattern` on the registry host in the template and disallow trailing slash and http(s):// prefix ?

Reply:
(not blocking, we can do that later)

Reply:
my suggestion is that any URLs should not have a trailing slash. it's the caller's responsibility to terminate the path. that's what we do in catalog.

@nl0 do you agree? you may have seen in slack that catalog is currently calling `https://url//some/path` -- it works but not pretty

Reply:
let's backlog that

Reply:
re `AllowPattern` - sure, but we should first enforce that in the schema itself, via [`pattern`](https://json-schema.org/understanding-json-schema/reference/regular_expressions.html)

Reply:
@akarve yep, i agree, and catalog expects `registryUrl` with no trailing slash

Reply:
I removed the trailing `/` from the autogenerated registryUrl